copier/questions/essential.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Essential questions
+package_name:
+  type: str
+  placeholder: my_python_package
+  help: Enter the name of the Python package.
+  validator: >-
+        {% if not (package_name | regex_search('^[a-z][a-z0-9\_]+$')) %}
+        Package name must start with a letter, followed one or more letters, digits or underscores all lowercase
+        {% endif %}
+version:
+  type: str
+  placeholder: 0.1.0
+  help: Enter the version of the Python package
```

Comment:
```suggestion
  help: Enter the version of the Python package.
```

Reply:
for the sake of uniformity I would like to keep it without full stop

Reply:
We do have some lines with a stop. Either would work!